EcoSoapBank/Network Controller/GraphQLController.swift

```diff
@@ -89,7 +89,7 @@ class GraphQLController: UserDataProvider, ImpactDataProvider, PickupDataProvide
     
     // Payments
     
-    func fetchPayments(forPropertyID propertyID: String, _ completion: @escaping ResultHandler<[Payment]>) {
+    func fetchPayments(forPropertyID propertyID: String, _ completion: @escaping (Result<[Payment], Error>) -> Void) {
```

Comment:
I would prefer this to be `ResultHandler` for consistency to match the other methods. It is a type alias, so will effectively be the same thing.

Reply:
Hmm, I'll take a look at that.